projects/plugins/jetpack/modules/publicize/publicize.php

```diff
@@ -531,7 +531,7 @@ public function show_options_popup( $service_name, $connection ) {
 		}
 
 		// if we have the specific connection info..
-		$id = sanitize_text_field( wp_unslash( $_GET['id'] ) ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		$id = ! empty( $_GET['id'] ) ? sanitize_text_field( wp_unslash( $_GET['id'] ) ) : false; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
Since `sanitize_text_field` typically returns a string, would it make sense to return an empty string here?

```suggestion
		$id = ! empty( $_GET['id'] ) ? sanitize_text_field( wp_unslash( $_GET['id'] ) ) : ''; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Reply:
Yes, good idea!